aisdc/attacks/likelihood_attack.py

```diff
@@ -32,6 +35,15 @@
 P_THRESH = 0.05  # default significance threshold
 
 
+def parse_boolean_argument(value: str) -> bool:
```

Comment:
in light of @rpreen comments -- is this method needed?

Reply:
Yes, removed the method given feedback from @rpreen